src/Snppts/Snippets/SimpleLogin.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using Snppts.Infrastructure;
+
+namespace Snppts.Snippets
+{
+    public class SimpleLogin : IAmASnippet
+    {
+        public string Slug => "simple-login";
+        public string Title => "Simple Login";
+        public string Description => "A simple login with option to continue as guest.";
+        public string GithubRepoName => "leijae/SimpleLogin";
+        public Uri ExternalUri { get; }
+        public bool ContainsAndroidSample => true;
+        public bool ContainsiOSSample => true;
+        public bool ContainsUWPSample => false;
+        public IAmAnAuthor AuthorInfo => new EricLeija();
+        public IEnumerable<Uri> ImageUris => new List<Uri> { new Uri("https://imgur.com/a/NbRLpp6") };
```

Comment:
Are you sure this URL will work as an image? Since I land directly on a webpage when going there?

It might be better to add the screenshot to your Github repo and link to that :)

Reply:
Thanks! Updated.